xamarin/Xamarin.Forms
# [Core] Added TextTransform property

### Description of Change ###

Added TextTransform property to the Label

### Issues Resolved ### 

- fixes #3667

### API Changes ###

Added:
 - TextTransform InputView.TextTransform { get; set; } //Bindable Property

```cs
Internals.TextTransform.Uppercase
Internals.TextTransform.Lowercase
Internals.TextTransform.None
```

### Platforms Affected ### 

- Core (all platforms)

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

- run Issue 3667
- click on button "Change TextTransform"
- check result

### PR Checklist ###
